src/pages/core/deserialization.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Deserialization
+metaTitle: Deserialization
+description: Learn about the deserialization of Asset accounts, Collection accounts and plugins.
+---
+
+Digital assets on Core are composed of exactly **one on-chain account** that has all the relevant data inside of it.
+
+That means that if we want to read that data we need to learn how to deserialize it.
+
+### Deserializing Assets
```

Comment:
Should we also add information about deserializing in Anchor in this guide?  Or save it for another guide? 
 i.e. suggesting that `Account<'info, BaseAssetV1>` can be used in Anchor (if you've tested it can be done that way).

Reply:
`Account<'info, BaseAssetV1>` isn't usable for me directly with Anchor account decoding. There's something in there blocking it still but looks like everything has Anchor Serialization and Deserialization on it in the children of the struct but it's still complaining that bot Anchor Serialization and Deserialization are not satisfied.

Reply:
Yeah per our offline discussion, we need to implement additional stuff to allow deserialization via Account struct.  I would say don't let it block this guide, but we will see if we can get it fixed soon.